### PR TITLE
feat(complex-table): translation for en-US

### PR DESCRIPTION
**Pull request by viztor**

### CODE REVIEW

File: docs/pattern/complex-table.en-US.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+<img class="preview-img" align="right" alt="Custom Columns" src="https://os.alipayobjects.com/rmsportal/zWUjQRpJZhYsZbY.png">
+
+Allow user to choose columns they want.
+
+<br>
+
+<img class="preview-img" align="right" alt="横向滚动示例" src="https://os.alipayobjects.com/rmsportal/UOTwZLweENvwlnL.png">
+
+Fixed header，view more by scroll vertically.
```
Comment (benjycui) [inline docs/pattern/complex-table.en-US.md]: `vertically` => `horizontally`

File: docs/pattern/complex-table.en-US.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<img class="preview-img" align="right" alt="二维表格示例" src="https://os.alipayobjects.com/rmsportal/RQvqwEkXpHaFago.png">
+
+A title vertically and horizontally each.
+
+### Mini-table
+
+<img class="preview-img" align="right" alt="小表格示例" src="https://os.alipayobjects.com/rmsportal/dMxxUThciHFQyXS.png">
+
+适用在卡片、弹出框等空间较小的场景中。
```
Comment (benjycui) [inline docs/pattern/complex-table.en-US.md]: Need to be deleted.

**Comment by viztor** [comment]
@benjycui fixed.

File: docs/pattern/complex-table.en-US.md (hunk 1 above)
Comment (viztor) [reply docs/pattern/complex-table.en-US.md]: fixed.

File: docs/pattern/complex-table.en-US.md (hunk 2 above)
Comment (viztor) [reply docs/pattern/complex-table.en-US.md]: fixed.

**Pull request merged**